src/modules/users/Data/api.js

```diff
@@ -38,6 +38,11 @@ export default {
     ${group}
     ${invite}
   `,
+  InviteUserNoWorkspace: gql`
```

Comment:
```suggestion
  InviteUserToPlatform: gql`
```

I think is a clearer name for the intent of this.